.github/workflows/deploy_pr_preview.yml

```diff
@@ -86,3 +87,36 @@ jobs:
               repo: context.repo.repo,
               body: output
             })
+  check_preview_links:
+    env:
+      BASE_URL: https://docs.egi.eu/documentation/users/
```

Comment:
In the preview, do we also want to extend this outside of `/users/`?

Reply:
definitely - if it's not done yet, I'll add it.